src/fragment-components/a-radiobutton-group.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { Dropdown,
+	TextInput,
+	RadioButtonGroup } from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+import image from './../assets/component-icons/radiobutton-group.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const ARadioButtonGroupStyleUI = ({ selectedComponent, setComponent }: any) => {
+	const orientationItems = [
+		{id: 'horizontal', text: 'Horizontal'},
+		{id: 'vertical', text: 'Vertical'}
+	];
+
+	const labelPositions = [
+		{id: 'left', text: 'Left'},
+		{id: 'right', text: 'Right'}
+	];
+	selectedComponent.valueSelected = selectedComponent.defaultSelected;
```

Comment:
🤔 Hmmm normally we would use a setter here, but if we do that, we would enter an infinite render loop? What is this for? I don't see valueSelected being used in this component at all, are you trying to initialize it & why here? What is the difference between value selected & default selected? S